docs/development/development.md

```diff
@@ -75,7 +75,7 @@ A `*` denotes a required variable.
 | Variable          | Value                             |  Notes     |
 |   :---            |   :---                            |    :---    |
 | `$BITOPS_DIR`     | `/opt/bitops`                     |   Within the container the default working directory for BitOps        |
-| `$ENVIRONMENT`*   | `YOUR_OPS_REPO_ENVIRONMENT`                   |   BitOps requires at least one environment folder to be specified at container execution.   |
+| `$BITOPS_ENVIRONMENT`*   | `YOUR_OPS_REPO_ENVIRONMENT`                   |   BitOps requires at least one environment folder to be specified at container execution.   |
```

Comment:
See https://github.com/bitovi/bitops/pull/269 (RFR too) which is fixing this in more places.